config/label_templates.yml

```diff
@@ -3,13 +3,13 @@ default_pmb_templates:
   :plate_double: plate_6mm_double_code39
   :plate_a: sqsc_96plate_label_template_code39
   :tube_rack: sqsc_96plate_label_template_code39
-  :tube: limber_tube_label_template
```

Comment:
limber_tube_label_template is mentioned in a couple of rake tasks and the print_jobs_controller.spec in Limber